### DISCUSSION TITLE
header image / background image?

### DISCUSSION THREAD

**Discussion by jorafb** [mmistakes/minimal-mistakes#3225] [Q&A]
Dear all, 

is there any way to add a header image? 

I found on the website that I could add something like :

header:
  image: /assets/images/image-filename.jpg
  
but I don't know exactly which file to modify.  I'm using an older version of the template :/

**Comment by iBug** [comment]
[The documentation](https://mmistakes.github.io/minimal-mistakes/docs/layouts/#headers) puts it very clearly. What about just giving it a try first?

**Reply by jorafb** [reply]
it does not work in the older version. Thanks anyway.

**Reply by mmistakes** [reply]
Looks like you're using a fork of the theme, so that could be why it's not working for you.
No idea if they have feature parity with Minimal Mistakes.

![Screen Shot 2021-10-29 at 12 15 24 PM](https://user-images.githubusercontent.com/1376749/139468525-7f790b74-8104-42e5-bae7-1ece45995373.png)

https://github.com/academicpages/academicpages.github.io
